src/sbvr-api/permissions.coffee

```diff
@@ -124,7 +125,31 @@ exports.setup = (app, sbvrUtils) ->
 		POSTPARSE: ({ req, request }) ->
 			# If the abstract sql query is already generated then adding permissions will do nothing
 			return if request.abstractSqlQuery?
+			if (request.method == 'POST' and request.odataQuery.property?.resource == 'canAccess')
+				request.odataQuery.options = { '$select': { 'properties': [ { 'name': 'id' } ] }, $top: 1 }
```

Comment:
This should really use the true id field (which may not be called `id`) via
```coffee
abstractSqlModel = sbvrUtils.getAbstractSqlModel(request)
abstractSqlModel.tables[request.resourceName].idField
```
once https://github.com/resin-io/pinejs/pull/106/files is merged

Reply:
Yeah that is a good point. When will #106 be merged? Should I wait for it?